x-pack/legacy/plugins/ml/public/jobs/new_job_new/recognize/components/create_result_callout.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, memo } from 'react';
+import { EuiCallOut, EuiButton, EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { SAVE_STATE } from '../page';
+
+interface CreateResultCalloutProps {
+  saveState: SAVE_STATE;
+  resultsUrl: string;
+  onReset: () => {};
+}
+
+export const CreateResultCallout: FC<CreateResultCalloutProps> = memo(
+  ({ saveState, resultsUrl, onReset }) => {
+    if (saveState === SAVE_STATE.NOT_SAVED) {
+      return null;
+    }
+    return (
+      <>
+        {saveState === SAVE_STATE.SAVED && (
+          <EuiCallOut
+            title={
+              <FormattedMessage
+                id="xpack.ml.newJob.recognize.jobsCreatedTitle"
+                defaultMessage="Jobs created"
+              />
+            }
+            color="success"
+            iconType="checkInCircleFilled"
+          />
+        )}
+        {saveState === SAVE_STATE.FAILED && (
+          <EuiCallOut
+            title={
+              <FormattedMessage
+                id="xpack.ml.newJob.recognize.jobsCreationFailed.saveFailedAriaLabel"
+                defaultMessage="Save failed"
+              />
+            }
+            color="danger"
+            iconType="alert"
+          />
+        )}
+        {saveState === SAVE_STATE.PARTIAL_FAILURE && (
+          <EuiCallOut
+            title={
+              <FormattedMessage
+                id="xpack.ml.newJob.recognize.someJobsCreationFailedTitle"
+                defaultMessage="Some jobs failed to be created"
+              />
+            }
+            color="warning"
+            iconType="alert"
+            aria-label={i18n.translate(
```

Comment:
Some of the Callouts in this fail have `aria-label`, some have not - is it necessary at all?